files/default/neutron-ha-tool.py

```diff
@@ -419,7 +425,7 @@ def l3_agent_migrate(qclient, noop=False, now=False):
     LOG.info("%d routers required migration from offline L3 agents",
              total_migrations)
     if total_errors > 0:
-        LOG.error("%d errors encountered during migration")
+        LOG.error("%d errors encountered during migration", total_errors)
```

Comment:
These were from #5 so I'm not sure why they are in this unrelated commit.  However I have now merged #5 so they will vanish on rebase.